### ISSUE TITLE
Storybook sb-padded-main NOT removed using "fullscreen" paramater

### ISSUE THREAD

**Issue by jlayden** [storybookjs/storybook#18138]
manager.js

```ts
addons.setConfig({
  isFullscreen: true,
});
```
preview.js

```ts
export const parameters = {
  layout: 'fullscreen',
};
```
Default padding is still being applied

**Comment by ndelangen** [comment]
Should work: https://storybook.js.org/docs/react/configure/story-layout#global-layout

Please supply a reproduction if the issue is still present in 7.0 beta.

**Comment by lkraav** [comment]
6.5.15: suddenly, with `webpack5` builder, I'm not able to get rid of padding anymore, unless I remove `--modern` build flag.

Weirdly, CI build with identical checkout works fine. Have tried eliminating `node_modules/.cache`, but no help. Completely stumped right now. :shrug:

**Comment by lkraav** [comment]
Got it fixed(?) in the weirdest way.

Ran `build-storybook --modern`, loaded it via `localhost` web server.

Now `start-storybook -p 6006 --modern` also works fine. I can edit `.storybook/preview.js` through all `layout` values, everything gets reflected instantly. This previously failed in all attempts.

I tested clearing browser caches, etc, nothing helped. No idea what's going on here, but maybe SB 7 will eliminate this problem permanently.
